FEEDBACK.md

```diff
@@ -24,14 +24,37 @@ Additional Reviews:
 
 
 ## Programs
+### Los Alamos Quantum Computing Summer School (ISTI)
+|Links|  |   |   |
+| :--: | :--: | :--: | :--: |
+
+Additional Reviews:
+
 ### QCSYS (University of Waterloo IQC)
 |Links|  |   |   |
 | :--: | :--: | :--: | :--: |
 
 Additional Reviews:
 
+
+### QOSF Mentorship Program (Quantum Open Source Foundation)
```

Comment:
This already exists in the repo, so this is creating a merge conflict. See above comment